dashboard.go

```diff
@@ -444,6 +445,13 @@ func UnmarshalViewPropertiesJSON(b []byte) (ViewProperties, error) {
 				return nil, err
 			}
 			vis = gv
+		case ViewPropertyTypeGeo:
```

Comment:
Im not super familiar with these models, but it appears there are some view marshalling tests:
`dashboard_test.go`
Could we add cases here to account for the new types? Just for good measure.

Reply:
Fixed, haven't noticed this since swagger generators do not complain about this problem.